qa/workunits/cephtool/test.sh

```diff
@@ -2093,6 +2093,14 @@ function test_mon_osd_tiered_pool_set()
   ceph osd pool create real-tier 2
   ceph osd tier add rbd real-tier
 
+  # expect us to be unable to set negative values for hit_set_*
+  for o in hit_set_period hit_set_count hit_set_fpp; do
+	expect_false ceph osd pool set real_tier $o -1
```

Comment:
i was about to complain. but, yike, just realized we already have 2-spaces indent, 4-spaces indent and tab indent in the same script. =)

Reply:
eh, that was actually masked by vim I think. Didn't realize that until you mentioned it. And now I gotta repush this so that I can sleep tonight ;)